clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -43,6 +43,8 @@
 #include "cache.h"
 #include "snoop_filter.h"
 
+enum cache_split_t { DATA, INSTRUCTION };
```

Comment:
Safer to put inside the class and not in the global namespace (xref #4343).  Or see other suggestions for the enum below.